react-app/src/History.js

```diff
@@ -1,11 +1,86 @@
-import { useState } from 'react'
+import { useState, useEffect } from 'react'
 import { StyleSheet, Text, View } from 'react-native'
 import { Calendar } from 'react-native-calendars'
 import NavBar from './NavBar.js'
 import SideMenu from './SideMenu.js'
+import * as SQLite from "expo-sqlite";
+
+
+function openDatabase() {
+  if (Platform.OS === "web") {
+    return {
+      transaction: () => {
+        return {
+          executeSql: () => {},
+        };
+      },
+    };
+  }
+
+  const db = SQLite.openDatabase("db.db");
+  return db;
+}
+
+const db = openDatabase();
 
 export default function History() {
+
   const [showMenu, setShowMenu] = useState(false)
+  const [debugMessage, setDebugMessage] = useState("debug2");
+
+  useEffect(() => {
+    db.transaction(
+      (tx) => {
+        // setDebugMessage('in tx2')
+        
+        tx.executeSql(
+          `create table if not exists sample_table (id integer primary key not null, name text);`,
```

Comment:
Check file "database\schemas\com.unicornio.happyinseat.database.MyDatabase\2.json" for the create statement we have been using.